docs/jep/11830-new-remat-checkpoint.md

```diff
@@ -0,0 +1,128 @@
+## `jax.remat` / `jax.checkpoint` changes: what you need to know
+
+
+#### Contents
+
+
+* [What's going on?](#what's-going-on)
+* [How can I disable the change, and go back to the old behavior for
+  now?](#-how-can-i-disable-the-change-and-go-back-to-the-old-behavior-for-now)
+* [Why are we doing this?](#why-are-we-doing-this)
+* [What are the possible issues after the upgrade?](#what-are-the-possible-issues-after-the-upgrade)
+
+
+
+#### What’s going on?
+
+As of [#11830](https://github.com/google/jax/pull/11830) we're switching on a new implementation of `jax.checkpoint`, aka `jax.remat` (the two names are aliases of one another). **For most code, there will be no changes.** But there may be some observable differences in edge cases; see [What are the possible issues after the upgrade?](#what-are-the-possible-issues-after-the-upgrade)
```

Comment:
Use ``` {func}`jax.checkpoint` ``` and ``` `{func}`jax.remat` ``` to get links to the functions in the rendered docs

Reply:
Nice, thanks!